### PR TITLE
Add an orientation option for swiper

### PR DESCRIPTION
**Pull request by shashankpandey2012**
Added an option for swiper orientation. User can keep swiper orientation horizontal or vertical. The default orientation is vertical as earlier.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +29,8 @@ new mapboxgl.Compare(before, after, {
 
 ```js
 compare = new mapboxgl.Compare(before, after, {
-  mousemove: true // Optional. Set to true to enable swiping during cursor movement.
+  mousemove: true, // Optional. Set to true to enable swiping during cursor movement.
+  orientation: 'horizontal' // Optional. Sets the orientation of swiper, defaults to vertical
```
Comment (andrewharvey) [inline README.md]: Given that vertical is the default and maybe more popular, and that many people just copy and paste examples, I think it makes more sense to set this to vertical.

File: index.js [hunk 2]
```diff
@@ -6,23 +6,25 @@ var EventEmitter = require('events').EventEmitter;
...
   this.options = options ? options : {};
+  this._horizontal = options.orientation === 'horizontal';
   this._onDown = this._onDown.bind(this);
   this._onMove = this._onMove.bind(this);
   this._onMouseUp = this._onMouseUp.bind(this);
   this._onTouchEnd = this._onTouchEnd.bind(this);
   this._ev = new EventEmitter();
   this._swiper = document.createElement('div');
-  this._swiper.className = 'compare-swiper';
+  this._swiper.className = this._horizontal ? 'swiper-horizontal'  : 'compare-swiper';
```
Comment (andrewharvey) [inline index.js]: could this be simpler if you used `compare-swiper-vertical` and `compare-swiper-horizontal`?

File: style.css [hunk 3]
```diff
@@ -20,3 +20,27 @@
...
   }
+
+.compare-horizontal {
```
Comment (andrewharvey) [inline style.css]: We should prefix these with `.mapboxgl-compare` to prevent issues with pages which map happen to use the same class.

Comment (shashankpandey2012) [reply style.css]: Done

File: index.js (hunk 2 above)
Comment (shashankpandey2012) [reply index.js]: Done

File: style.css [hunk 4]
```diff
@@ -20,3 +20,25 @@
...
   }
+
+.horizontal {
```
Comment (andrewharvey) [inline style.css]: This should be prefixed otherwise anyone who includes this swiper on a page which also uses .horizontal will break their css.

Comment (shashankpandey2012) [reply style.css]: Done

Comment (pandeyshashank) [reply style.css]: Any more changes is required @andrewharvey ?

**Comment by andrewharvey** [comment]
Looks good to me @shashankpandey2012 thanks. There were some formatting/style issues, but to be honest it passes lint so it's okay. In the future if we use a stricter lint these can be changed.

**Comment by andrewharvey** [comment]
I'll leave it open a few more days for anyone else to review, if no other comments I can merge it.

**Comment by shashankpandey2012** [comment]
> I'll leave it open a few more days for anyone else to review, if no other comments I can merge it.

Thanks

**Pull request merged**